ONSdigital/dp-dimension-extractor
# Feature/stop dp dimension extractor killing itself

### What

Now the service doesn't die if external services don't run, or if validation fails. If services die during this service executions, it does not die either.

### How to review

- Check that you can run the service without Kafka, and it does not die (it logs the errors).
- Try to start the service with Kafka running, then kill Kafka, and the service does not die (it logs the errors).
- Try to start the service with an invalid token or no token, and it should not die.
- The service should continue to function as expected when the rest of the services run.

### Who can review

Anyone
